src/features/DecisionAids/Recommendations/index.js

```diff
@@ -89,6 +90,8 @@ function Recommendations(props) {
         </Card.Body>
       </Card>
       <RiskEstimates input={riskTable} />
+      <br/>
+      <p>CDS Apply Time: {applyTime}ms </p>
```

Comment:
This was part of an earlier branch that added the CDS Apply time to the bottom of the Recommendation as an aid to performance debugging. Do we still want to include this?